src/main/java/appeng/core/AppEng.java

```diff
@@ -189,14 +190,14 @@ private void handleIMCEvent( FMLInterModComms.IMCEvent event )
 	@EventHandler
 	private void serverStopping( FMLServerStoppingEvent event )
 	{
-		WorldSettings.getInstance().shutdown();
+		WorldData.onServerAboutToStart();
```

Comment:
`onServerAboutToStart` for the stopping event? Is that either incorrectly named or the wrong place?